src/filter/TargetingFilter.ts

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IFeatureFilter } from "./FeatureFilter";
+import { createHash } from "crypto";

```

Comment:
The portal team used CryptoJS. I imagine either works- but might make sense to align on one https://msazure.visualstudio.com/Azure%20AppConfig/_git/Portal?path=/AppConfigPortalExtension/AppConfigPortalExtension/Client/Resource/FeatureManager/ResourceFeatureFlagsBlade.ts&version=GBmaryanne/feature/splitExperimentation&_a=contents